standup/settings.py

```diff
@@ -301,6 +284,15 @@ def path(*paths):
             ),
             'output_filename': 'js/common.min.js',
         },
+        'signin': {
+            'source_filenames': (
+                'js/vendor/jquery-3.1.0.js',
+                'js/vendor/lock-passwordless-2.2.min.js',
```

Comment:
lock-passwordless-2.2.min.js is biggish, so I figured it made sense to keep a separate signin page so we didn't have to download it with other pages.

Yes, we have caching and all that, but seemed like a better idea to just keep other pages minimal.